data-access-analysis-tools/index-portal-intro.rst

```diff
@@ -59,7 +61,7 @@ Keeping the search area to a minimum will keep processing times short and return
 
```

Comment:
In the above section, two small suggestions:
"right ascencion" --> "right ascension", and "The examples under the box for coordinates are object names are examples of the formatting only." --> "The examples under the box for coordinates are object names as examples of the formatting only."